### PR TITLE
Update the introduction of UCASE() and UNHEX()

### PR DESCRIPTION
**Pull request by heywalter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the introduction of UCASE() and UNHEX(), fix https://github.com/pingcap/docs-cn/issues/15952

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/string-functions.md [hunk 1]
```diff
@@ -308,11 +308,47 @@ SELECT BIN("123q123");
...
+
+查询示例
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
+```
+
+返回结果
+
+```sql
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:325]: 返回值请写成下面的形式：
```sql
    +--------------------------+
    | approx_percentile(a, 50) |
    +--------------------------+
    |                        2 |
    +--------------------------+
```

File: functions-and-operators/string-functions.md [hunk 2]
```diff
@@ -308,11 +308,47 @@ SELECT BIN("123q123");
...
 ### [`UCASE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ucase)
 
-与 `UPPER()` 功能相同
+`UCASE()` 函数将字符串转换为大写字母，此函数等价于 `UPPER()` 函数。
+
+> **注意：**
+>
+> 当字符串为 `NULL` 时，则返回 `NULL`。
+
+查询示例
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:317]: 请加上冒号

File: functions-and-operators/string-functions.md [hunk 3]
```diff
@@ -308,11 +308,47 @@ SELECT BIN("123q123");
...
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
+```
+
+返回结果
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:323]: 请加上冒号

File: functions-and-operators/string-functions.md (hunk 1 above)
Comment (heywalter) [reply functions-and-operators/string-functions.md:325]: When executing SQL, it returns as follows: The format does not match yours, which is correct?
image

Comment (heywalter) [reply functions-and-operators/string-functions.md:325]: I've updated the SQL results. It looks like a compatibility issue of tiup client, MySQL Client vs Tiup Client as below:

image

File: functions-and-operators/string-functions.md [hunk 4]
```diff
@@ -308,11 +308,51 @@ SELECT BIN("123q123");
...
+| result_upper | result_null |
++--------------+-------------+
+| BIGDATA      | NULL        |
++--------------+-------------+
+```
 
 ### [`UNHEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_unhex)
 
-返回一个数的十六进制表示，形式为字符串
+执行 `HEX()` 函数的逆运算，将参数中的每对字符视为十六进制数字，并将其转换为该数字表示的字符，返回值为二进制字符串。
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:335]:
```suggestion
`UNHEX()` 函数执行 `HEX()` 函数的逆运算，将参数中的每对字符视为十六进制数字，并将其转换为该数字表示的字符，返回值为二进制字符串。
```

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**